src/main.py

```diff
@@ -40,7 +43,18 @@ def main() -> None:
     """
 
     while True:
-        get_input()
+        #get imput 
+        for event in pg.event.get():
```

Comment:
To keep the code modular, the following input functionality should be placed in a separate function